eclair-node/src/main/scala/fr/acinq/eclair/Boot.scala

```diff
@@ -39,7 +43,29 @@ object Boot extends App with Logging {
     val setup = new Setup(datadir)
     plugins.foreach(_.onSetup(setup))
     setup.bootstrap onComplete {
-      case Success(kit) => plugins.foreach(_.onKit(kit))
+      case Success(kit) =>
+        plugins.foreach(_.onKit(kit))
+        val config = NodeParams.loadConfiguration(datadir, ConfigFactory.empty())
```

Comment:
It's a bit weird to have to reload the configuration, could we had the relevant parts to `Kit` instead?
Maybe that is tied to a bigger config refactoring and should be done later though.

Reply:
Agreed, in commit 3469eb5 i don't reload the config anymore but directly access the field on the Setup class.

Reply:
Agreed, in commit 3469eb5 i don't reload the config anymore but directly access the field on the Setup class.